packages/transactions/src/serializers/__tests__/address-test.ts

```diff
@@ -0,0 +1,70 @@
+import { Base58EncodedAddress } from '@solana/keys';
+
+import { getBase58EncodedAddressCodec } from '../address';
+
+describe('Base58 encoded address codec', () => {
+    let address: ReturnType<typeof getBase58EncodedAddressCodec>;
+    beforeEach(() => {
+        address = getBase58EncodedAddressCodec();
+    });
+    it('serializes a base58 encoded address into a 32-byte buffer', () => {
+        expect(
+            address.serialize(
+                '4wBqpZM9xaSheZzJSMawUHDgZ7miWfSsxmfVF5jJpYP' as Base58EncodedAddress<'4wBqpZM9xaSheZzJSMawUKKwhdpChKbZ5eu5ky4Vigw'>
+            )
+        ).toMatchInlineSnapshot(`
```

Comment:
Just wondering, why does this use `toMatchInlineSnapshot` instead of toBe/toEqual?

Reply:
Yeah, probably more appropriate to do what you say, here!